tests/test_expressions.py

```diff
@@ -243,6 +243,17 @@ def test_replace_tables(self):
             'SELECT * FROM a1 AS a /* a */, b.a /* b */, c.a2 /* c.a */, d2 /* d.a */ CROSS JOIN e.a CROSS JOIN "F" /* f-F.A */ CROSS JOIN g1.a /* g */',
         )
 
+        self.assertEqual(
+            exp.replace_tables(
+                parse_one("select * from example.table", dialect="bigquery"),
+                {
+                    "example.table": "`my-project.example.table`"
+                },
+                dialect="bigquery",
+            ).sql(),
+            'SELECT * FROM "my-project".example.table /* example.table */',
```

Comment:
This is invalid in BQ dialect. "my-project" should be wrapped in backticks - `my-project`